tasks/src/zero-shot-classification/about.md

```diff
@@ -0,0 +1,20 @@
+## Use Cases
+
+### Emotion Analysis
+
+Zero-shot emotion analysis is the task of analyzing the emotions of people in images, without any training data for those emotions. This can be done by using a pre-trained model that has been trained on a large dataset of images with labels for different emotions and then using that model to predict the emotions of people in new images.
```

Comment:
```suggestion
Zero-shot sentiment analysis is the task of analyzing the sentiments of people in the text, without any training data for those sentiments. This can be done by using a model that was previously trained on a large dataset of images with labels for different sentiments and then using that model to predict the sentiments of people in new images.
```
Why images? This task is for text, no? I also don't see how it's zero shot given the model has already seen the sentiments previously (as you've written later on).